libqtile/utils.py

```diff
@@ -219,22 +230,19 @@ def class_proxy(*args, **kwargs):
         globals_[class_name] = class_proxy
 
 
-def send_notification(title, message, urgent=False, timeout=10000):
+def send_notification(title, message, urgent=False, timeout=10000, id=None):
     """Send a notification."""
-    try:
-        import gi
-        gi.require_version("Notify", "0.7")
-        from gi.repository import Notify
-        Notify.init("Qtile")
-        info = Notify.get_server_info()
-        if info[0]:
-            notifier = Notify.Notification.new(title, message)
-            notifier.set_timeout(timeout)
-            if urgent:
-                notifier.set_urgency(Notify.Urgency.CRITICAL)
-            notifier.show()
-    except Exception as exception:
-        logger.error(exception)
+    if _can_notify and Notify.get_server_info()[0]:
+        notifier = Notify.Notification.new(title, message)
+        notifier.set_timeout(timeout)
+        if urgent:
+            notifier.set_urgency(Notify.Urgency.CRITICAL)
+        notifier.set_timeout(timeout)
```

Comment:
Why are you setting the timeout twice (four lines above it is also set)? If there is a reason, a comment would be important.

Reply:
Nope, that is a total typo! Thanks for spotting it.